assets/js/pages/ActivityLogPage/ActivityLogPage.jsx

```diff
@@ -83,6 +120,31 @@ function ActivityLogPage() {
             setActivityLogDetailModalOpen(false)
           }
         />
+        <PaginationPrevNext
```

Comment:
**question:** what made you opt for placing this component here and not in `ActivityLogOverview` after the table?‏
I see it fitting there better.

Reply:
I thought the same before I started to develop. Then I realized that actions on the pagination trigger HTTP requests that must be intercepted and executed at the page level. If we put pagination inside `ActivityLogOverview` we need to bubble up events and have a _prop-drilling_ situation.

Reply:
Yes, indeed we'd be passing props to `ActivityLogOverview` that will be just then passed down to `PaginationPrevNext`.

I am not sure what is _less bad_: prop-drilling or having `PaginationPrevNext` here :smile: 
Up to you, no strong opinions.

What would you do to mitigate prop-drilling and still have the `PaginationPrevNext` next to the table?
React contexts? HOCs? Other tecniques?

Reply:
I see no evil in leaving the pagination at page level, to be honest. The purpose of having `ActivityLogOverview` is to have unit tests on that, and adding pagination won't allow us to add any meaningful test imho - we should just test "the prop is assigned to the component" or replicate things already tested in the pagination component itself.